packages/sdk/utils/fetch.ts

```diff
@@ -47,8 +47,12 @@ const errorHandler = <T>(fetch: Fetch<T>): Fetch<T> => {
       return response
     } else {
       const body = await response.json()
-      const errorMessage = body?.error?.message || body?.message || JSON.stringify(body)
-      throw new DfnsError(response.status, errorMessage, body.error)
+      const message = body?.error?.message ?? body?.message
+      throw new DfnsError(response.status, message, {
```

Comment:
did a quick test, we bubble up the `FetchError`, only throw `DfnsError` when it's from the server side.

```
> ts-node main.ts

/Users/danieljin/Projects/dfns/dfns-sdk-ts/node_modules/node-fetch/lib/index.js:1501
			reject(new FetchError(`request to ${request.url} failed, reason: ${err.message}`, 'system', err));
          ^
FetchError: request to https://api.dfns.wt/auth/action/init failed, reason: getaddrinfo ENOTFOUND api.dfns.wt
    at ClientRequest.<anonymous> (/Users/danieljin/Projects/dfns/dfns-sdk-ts/node_modules/node-fetch/lib/index.js:1501:11)
    at ClientRequest.emit (node:events:513:28)
    at ClientRequest.emit (node:domain:489:12)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at TLSSocket.emit (node:domain:489:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) {
  type: 'system',
  errno: 'ENOTFOUND',
  code: 'ENOTFOUND'
}
```

vs

```
DfnsError: Wallet not found
    at /Users/danieljin/Projects/dfns/dfns-sdk-ts/packages/sdk/utils/fetch.ts:51:13
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async WalletsClient.getWallet (/Users/danieljin/Projects/dfns/dfns-sdk-ts/packages/sdk/generated/wallets/client.ts:132:22)
    at async main (/Users/danieljin/Projects/dfns/dfns-sdk-ts/examples/sdk/service-account/main.ts:24:13) {
  httpStatus: 404,
  context: {
    id: '5334229225287731691',
    status: 404,
    message: 'Wallet not found',
    details: { walletId: 'fooo', orgId: 'or-yanke-mars-6ulofamogg84s87v' },
    debugTrace: 'Error: Wallet not found\n' +
      '    at Function.notFound (/libs/shared-libs/src/errors/index.ts:121:12)\n' +
      '    at getOrgWallet (/services/wallets/dockers/wallets-api/src/utils/getOrgWallet.ts:12:21)\n' +
      '    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n' +
      '    at getWallet (/services/wallets/dockers/wallets-api/src/api/getWallet.ts:16:18)'
  }
}
```